### PR TITLE
Update rails new.

### PR DESCRIPTION
**Pull request by nathancolgate**
Updating this to use some of the new latest/greatest config.

### CODE REVIEW

File: rails-new.md [hunk 1]
```diff
@@ -21,19 +21,21 @@ In your new directory make a `docker-compose.yml` file and paste in:
...
+      - DATABASE_URL=postgres://postgres:monkey@postgresservice:5432
     volumes:
       - .:/app:cached
       - bundle_cache:/usr/local/bundle
     ports:
       - "3000:3000"
     depends_on:
-      - postgres
-  postgres:
+      - postgresservice
```
Comment (kmdunn5) [inline rails-new.md:34]: I haven't seen `postgresservice`. Is this doing the same?  I'm also assuming the password doesn't matter here, but why monkey?

**Review by kmdunn5 — APPROVED** [review]
LGTM other than the comment

File: rails-new.md (hunk 1 above)
Comment (nathancolgate) [reply rails-new.md:34]: This is intentional. Mostly trying to help folks parse what's going on. 

Previously: the service, the username, the actual name of the application were all the same: `postgres`

By introducing differences here i'm trying to help first-time users map between things. Like "Oh... postgresservice is in two places... they must map to each other. And password isn't just password.. it's monkey, which is set down here."

**Comment by willtcarey** [comment]
If I remember correctly I had locked Rails to the 6.x release because 7.0 stopped shipping webpacker and the rest of the guide makes the assumption that you have webpacker (have I mentioned how much I dislike Rail's changing asset pipelines in every release?)

I think we can just rewrite the setup process in the stimulus chapter to install jsbundling-rails and the rest of the guide should proceed as normal. https://github.com/brandnewbox/bnb-on-rails-book/issues/66

**Pull request merged**